keyboards/lime/readme.md

```diff
@@ -0,0 +1,30 @@
+# Lime Keyboard
+
+![LimeKeyboard version 1](https://www.simonepellegrino.com/)
+
+Lime is 6×4+6 keys column-staggered split keyboard. Based on Sofle with a hint of Lily58, Corne and Helix keyboards.
+
+Build guide and a build log not available yet
+
+* Keyboard Maintainer: [HellSingCoder](https://www.simonepellegrino.com/) [GitHub: HellSingCoder](https://github.com/HellSingCoder)  
+* Hardware Supported: LimeKeyboard PCB, ProMicro  
+* Hardware Availability: [PCB & Case Data](https://github.com/HellSingCoder/LimeKeyboard)
+
+Make example for this keyboard (after setting up your build environment):
+
+    make lime:default
+    
+        or
+        
+    qmk compile -kb lime -km default
+    qmk flash -kb lime -km default
+
+Flash the default keymap: 
+
+    make lime:default:avrdude
+
+Press reset button on he keyboard when asked.
+
+Disconnect the first half, connect the second one and repeat the process.
```

Comment:
Could you add instructions on how to enter the bootloader, to flash new firmware? 

Eg: 
```markup
## Bootloader

Enter the bootloader in 3 ways:

* **Bootmagic reset**: Hold down the key at (0,0) in the matrix (usually the top left key or Escape) and plug in the keyboard
* **Physical reset button**: Briefly press the button on the back of the PCB - some may have pads you must short instead
* **Keycode in layout**: Press the key mapped to `RESET` if it is available
```